msg/de_DE.utf8/utils.msg

```diff
@@ -1232,7 +1232,7 @@ gültigen Optionen:\n\
   -e, --exclude-class-file=FILE Eingabedatei der Tabellennamen, um auszuschließen; entweder -i oder -e,\n\
                                 verwendet werden, jedoch nicht beides\n\
   -t, --timeout=NUMBER          Abfrage-Timeout in Millisekunden; Standard: 1000 ms\n\
-  -n, --table-name=NAME         Tabellennamen, um Prüfsumme Ergebnisse zu speichern; Standard: db_ha_checksums\n\
+  -n, --table-name=NAME         Tabellennamen, um Prüfsumme Ergebnisse zu speichern; Standard: db_ha_checksum\n\
```

Comment:
Shouldn't this also be changed to dba.db_ha_checksum?

Reply:
I think default value of class is better that `db_ha_checksum`. 

Because I think it is similar normal table name. 
(There is no description, default table owner  is 'public' . )The verification of owner is up to the user.

Reply:
I think it is essential to give accurate information through help messages. Depending on who owns the table, there can be more than one table named db_ha_checksum, and I think it is necessary to tell which table the information is stored in. Even though the help message can provide enough information, I don't understand why the user has to find out which table the information is stored in with only the name db_ha_checksum.

> Because I think it is similar normal table name.
> (There is no description, default table owner is 'public' . )
I don't understand what this means

Reply:
I don't know why you are looking for the default owner. Owner means the subject who created the table. Since csql was executed without the -u option, csql was executed as a public user, and since the public user created a table, the owner of the table only became public.

Reply:
And users don't know that utility is executed by the dba user.

Reply:
It would be nice to follow @joohok's suggestion. In this approach, tables with the same name but different owners can exist. If the usages doesn't describe the owner of the db_ha_checksum, the user doesn't know which db_ha_checksum table they're talking about.

Reply:
if you change this to dba.db_ha_checksum, you also need to edit the manual (https://www.cubrid.org/manual/en/11.0/ha.html#checksumdb). 

* description of the option `--table-name`
* examples

Reply:
Thank you I will apply it. :)